### PR TITLE
Created checkbox component

### PR DESCRIPTION
**Pull request by dpionk**

### CODE REVIEW

**Review by kbojarowska — CHANGES_REQUESTED** [review]
Checkbox is too big, the check icon is too small
"checked" props doesn't work

**Comment by kbojarowska** [comment]
If it's possible please use fi icons from react icons gallery

File: skin-resque-frontend/src/components/Checkbox/Checkbox.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+import PropTypes from 'prop-types';
+import Text from '../Text/Text';
+import './Checkbox.scss';
+
+function Checkbox({checked, children}) {
+	const [isChecked, setIsChecked] = useState(checked);
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/components/Checkbox/Checkbox.js:7]: isChecked not used

File: skin-resque-frontend/src/components/Checkbox/Checkbox.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+	return (
+		<div className='checkbox-wrapper'>
+			<input type='checkbox' onChange={() => setIsChecked((checked) => !checked)}/>
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/components/Checkbox/Checkbox.js:11]: Input doesn't use isChecked for its checked property

File: skin-resque-frontend/src/components/Checkbox/Checkbox.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+			<Text>{children}</Text>
+		</div>
+	);
+};
+
+export default Checkbox;
+
+Checkbox.propTypes = {
+  checked: PropTypes.bool
+};
```
Comment (MrSpaceMan1) [inline skin-resque-frontend/src/components/Checkbox/Checkbox.js:21]: Lack of children type

**Review by MrSpaceMan1 — CHANGES_REQUESTED** [review]

**Comment by dpionk** [comment]
@kbojarowska  @MrSpaceMan1  I made changes after review, I tried using react icons but it was hard for me as I could not find on internet how to do it and had no idea

**Review by MrSpaceMan1 — APPROVED** [review]

**Review by kbojarowska — APPROVED** [review]

**Pull request merged**